proposals/shared-everything-threads/Overview.md

```diff
@@ -116,6 +116,20 @@ meaning `shared` or `unshared` introduced in the shared memory proposal.
 
 Shared memories have already been standardized.
 
+For consistency with other shared annotations, the existing text format for shared memories is
+redefined to be an abbreviation of the new text format:
+
+```
+memtype ::= it:index_type lim:limits => unshared it lim
+          | 'shared' it:index_type lim:limits => shared it lim
+
+'(' 'memory' id? index_type limits 'shared' ')'
+  == '(' 'memory' id? 'shared' index_type limits ')'
+
+'(' 'memory' id? '(' 'import' name1 name2 ')' index_type limits 'shared' ')'
+  == '(' 'memory' id? '(' 'import' name1 name2 ')' 'shared' index_type limits ')'
```

Comment:
I think you can simplify and unify these two by defining it as an abbreviation for memory types:
```suggestion
index_type limits 'shared'
  == 'shared' index_type limits
```
Then it also automatically applies to unsugared imports, which is missing.

Reply:
Right, makes sense. I had the version covering the whole memory because of my misunderstanding about how abbreviations are expanded.